### PR TITLE
Update step-2-configure-a-two-way-wcf-webhttp-send-port.md

### PR DESCRIPTION
**Pull request by msshaheera**
Corrected extra space and missing curly brackets in the Tip for URL Mapping.  Also added comments and example about double-escaping when the REST service requires xml-escaped characters in the URL.

### CODE REVIEW

**Comment by msshaheera** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 1]
```diff
@@ -45,12 +45,21 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
          > [!TIP]
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:45]:
```suggestion
         > [!IMPORTANT]
         >

```
```suggestion
         > [!TIP]
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 2]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
 
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
 
-         > [!TIP]
+         > [!IMPORTANT]
+         >
+
          > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:48]:
```suggestion
         > Within the URL field, you must "escape" any special XML characters to make sure that the port 
         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 3]
```diff
@@ -42,15 +42,27 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
 
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port 
+         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+           >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:50]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 4]
```diff
@@ -42,15 +42,27 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
 
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port 
+         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+           >For:
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:51]:
```suggestion
         > For the following text:
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 5]
```diff
@@ -42,15 +42,27 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port 
+         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+           >`Url=”/Customer?{ID}&group={Location}”`
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:52]:
```suggestion
         > `Url="/Customer?{ID}&group={Location}"`
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 6]
```diff
@@ -42,15 +42,27 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
 
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
 
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:47]:
```suggestion
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 7]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port 
+         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:52]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 8]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
 
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
 
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:47]:
```suggestion
         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 9]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
 
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
 
-         > [!TIP]
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > processes and preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:48]:
```suggestion
         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 10]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
-         > Within the URL field, any special XML characters must be "escaped". This ensures that the special XML characters are processed, and preserved by the port. For example, the `&` special character must be escaped as `&amp;`.
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:53]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 11]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+           >Use:
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:54]:
```suggestion
         >Use the following text instead: 
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 12]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+           >`Url=”/Customer?{ID}&amp;group={Location}”`
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:55]:
```suggestion
         > `Url="/Customer?{ID}&amp;group={Location}"`
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 13]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:56]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 14]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires the special character be escaped within the URL used at runtime, double-escaping may be required.
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:57]:
```suggestion
         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 15]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:58]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 16]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+           >For:
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:59]:
```suggestion
         > For the following text: 
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 17]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+           >`Url=”/Customer?{ID}&amp;group={Location}”`
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:60]:
```suggestion
         > `Url="/Customer?{ID}&amp;group={Location}"`
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 18]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         >
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:61]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 19]
```diff
@@ -42,15 +42,26 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
 
          Here, **GET** is the HTTP verb and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:43]:
```suggestion
         Here, **GET** is the HTTP verb, and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 20]
```diff
@@ -40,17 +40,28 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         >
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
            >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:62]:
```suggestion
         >
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 21]
```diff
@@ -40,17 +40,28 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > Use the following text instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         >
+           >Use:
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:63]:
```suggestion
         > Use the following version instead: 
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 22]
```diff
@@ -40,17 +40,28 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
+         >
+         > Use the following text instead:
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:54]:
```suggestion
         > Use the following version instead: 
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 23]
```diff
@@ -40,17 +40,28 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         >
+         > Use the following version instead: 
+           >`Url=”/Customer?{ID}&amp;amp;group={Location}”`
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:64]:
```suggestion
         > `Url="/Customer?{ID}&amp;amp;group={Location}"`
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 24]
```diff
@@ -40,17 +40,28 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         >
+         >
+         > Use the following version instead: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
+         > If the REST service also requires that you escape the special character within the URL used at runtime, you might have to use double-escaping.
+         >
+         > For the following text: 
+         > `Url="/Customer?{ID}&amp;group={Location}"`
+         >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:61]:
```suggestion
```

File: biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md [hunk 25]
```diff
@@ -40,17 +40,27 @@ In this step you configure a two-way **WCF-WebHttp** send port to invoke the RES
...
+         Here, **GET** is the HTTP verb, and **On_Time_Performance** is appended to the base URI to construct a unique resource URL to retrieve flight delays.
+
+         > [!IMPORTANT]
+         >
+         > Within the URL field, you must "escape" any special XML characters to make sure that the port processes and 
+         > preserves the special characters. For example, you must escape the `&` special character as `&amp;`.
+         >
+         > For the following text:
+         > `Url="/Customer?{ID}&group={Location}"`
+         >
```
Comment (ecfan) [inline biztalk/core/step-2-configure-a-two-way-wcf-webhttp-send-port.md:52]:
```suggestion
```

**Review by ecfan — APPROVED** [review]

**Pull request merged**